frontend/src/components/UserItem/index.tsx

```diff
@@ -11,13 +11,22 @@ type User = {
   nickname: string;
   accessToken: null | string;
   tardyCount: number;
+  attendanceStatus: string;
 };
 
 type UserItemProps = {
   user: Omit<User, 'accessToken'>;
+  onAttendanceCheck: (
+    user: Omit<User, 'accessToken'>,
+    checked: boolean
+  ) => void;
 };
 
-const UserItem: React.FC<UserItemProps> = ({ user }) => {
+const UserItem: React.FC<UserItemProps> = ({ user, onAttendanceCheck }) => {
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    onAttendanceCheck(user, e.target.checked);
+  };
```

Comment:
로딩 시 optimisticChecked 상태를 보여주면 요구사항을 충족시킬 수 있을 것 같습니다.

```suggestion
const UserItem: React.FC<UserItemProps> = ({ user, loading, onAttendanceCheck }) => {
  const [optimisticChecked, setOptimisticChecked] = useState<boolean>(user.attendanceStatus);

  const handleChange: React.FormEventHandler<HTMLInputElement> = (e) => {
    setCurrentChecked(e.target.checked)
    onAttendanceCheck(user, e.target.checked);
  };
```

Reply:
meeting page에서 userItem으로 check핸들러를 이동했습니다!